content/updates/04-2020/2-associations-updates.md

```diff
@@ -0,0 +1,30 @@
+# Product associations update
+
+
+## View the number of associations per product
+::: meta-data type="New" features="Product associations, Productivity, Data Governance" available="10th of April" link-to-doc="../articles/products-associations.html#view-the-products-associations"
+:::
+
+We now display the **total number of associations per product or product model nearby the `Associations` tab** when you are your product form.    
```

Comment:
"when you are in your product form."
The "in" was missing here.

Reply:
I put 'on your product form' @camillebourjade is this correct?